lesson4/Homework/Tasks.js

```diff
@@ -0,0 +1,79 @@
+var personnel = [{
+        id: 5,
+        name: "Luke Skywalker",
+        pilotingScore: 98,
+        shootingScore: 56,
+        isForceUser: true,
+    },
+    {
+        id: 82,
+        name: "Sabine Wren",
+        pilotingScore: 73,
+        shootingScore: 99,
+        isForceUser: false,
+    },
+    {
+        id: 22,
+        name: "Zeb Orellios",
+        pilotingScore: 20,
+        shootingScore: 59,
+        isForceUser: false,
+    },
+    {
+        id: 15,
+        name: "Ezra Bridger",
+        pilotingScore: 43,
+        shootingScore: 67,
+        isForceUser: true,
+    },
+    {
+        id: 11,
+        name: "Caleb Dume",
+        pilotingScore: 71,
+        shootingScore: 85,
+        isForceUser: true,
+    },
+];
+
+const countries = [
+    { "name": "Afghanistan", "code": "AF" },
+    { "name": "Åland Islands", "code": "AX" },
+    { "name": "Albania", "code": "AL" },
+    { "name": "Algeria", "code": "DZ" }
+];
+
+// Task 1
+function getCountryByCode(code) {
+    return countries.find(country => country.code == code);
```

Comment:
Use strict comparison `===` also if country is found, it should return its name. Now it returns object `{ "name": "Algeria", "code": "DZ" }`